dev/main.sh

```diff
@@ -12,8 +12,56 @@ echo "====> ✅Step 1: finish preparation"
 # Run docker compose
 echo $DIVIDER
 echo "====> 👀Step 2: start calling docker compose up"
-docker-compose -f docker-compose-sdp-anchor.yml down && docker-compose -f docker-compose-sdp-anchor.yml up --abort-on-container-exit
+docker-compose down && docker-compose -p sdp-multi-tenant up -d --build
 echo "====> ✅Step 2: finish calling docker-compose up"
 
+# Initialize tenants
 echo $DIVIDER
-echo "🎉🎉🎉🎉 SUCCESS! 🎉🎉🎉🎉"
+echo "====> 👀Step 3: initialize tenants"
```

Comment:
do we wanna do something like
if ! command -v curl &> /dev/null; then
    echo "Error: curl is not installed." >&2
    exit 1
fi
at the beginnign of the script?

Reply:
Addressed, thank you!